lib/android/app/src/main/java/com/reactnativenavigation/parse/TopBarOptions.java

```diff
@@ -17,43 +17,57 @@ public static TopBarOptions parse(TypefaceLoader typefaceManager, JSONObject jso
 		TopBarOptions options = new TopBarOptions();
 		if (json == null) return options;
 
-		options.title = json.optString("title", NO_VALUE);
-		options.backgroundColor = json.optInt("backgroundColor", NO_COLOR_VALUE);
-		options.textColor = json.optInt("textColor", NO_COLOR_VALUE);
-		options.textFontSize = (float) json.optDouble("textFontSize", NO_FLOAT_VALUE);
-		options.textFontFamily = typefaceManager.getTypeFace(json.optString("textFontFamily", NO_VALUE));
-		options.hidden = NavigationOptions.BooleanOptions.parse(json.optString("hidden"));
-		options.animateHide = NavigationOptions.BooleanOptions.parse(json.optString("animateHide"));
+        options.title = json.optString("title", NO_VALUE);
+        options.backgroundColor = json.optInt("backgroundColor", NO_COLOR_VALUE);
+        options.textColor = json.optInt("textColor", NO_COLOR_VALUE);
+        options.textFontSize = (float) json.optDouble("textFontSize", NO_FLOAT_VALUE);
+        options.textFontFamily = typefaceManager.getTypeFace(json.optString("textFontFamily", NO_VALUE));
+        options.hidden = NavigationOptions.BooleanOptions.parse(json.optString("hidden"));
+        options.animateHide = NavigationOptions.BooleanOptions.parse(json.optString("animateHide"));
+        options.collapse = NavigationOptions.BooleanOptions.parse(json.optString("hideOnScroll"));
```

Comment:
Consider renaming collapse to hideOnScroll? Having the same names across Js and native helps a lot